src/pages/BookDetails/BookDetails.js

```diff
@@ -1,7 +1,31 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import bookIMG from "../../assets/images/book_thumbnail.jpeg"
+import Typography from "@mui/material/Typography";
+
 import classes from "./BookDetails.module.scss";
```

Comment:
order of imports should go like this:
- external library imports
- imports from outside the current directory
- imports from the working directory

so the import would look like this:
```suggestion
import * as React from 'react';
import Box from '@mui/material/Box';
import Grid from '@mui/material/Grid';
import Typography from "@mui/material/Typography";

import bookIMG from "assets/images/book_thumbnail.jpeg"

import classes from "./BookDetails.module.scss";
```

why `"assets/images/book_thumbnail.jpeg"` and not `"../../assets/images/book_thumbnail.jpeg"`? We can add absolute imports to the project to clean up our code: [tutorial](https://medium.com/geekculture/making-life-easier-with-absolute-imports-react-in-javascript-and-typescript-bbdab8a8a3a1) there is even a webStorm configuration there so that WebStorm knows we are using absolute imports.